scripts/markets/create_markets.js

```diff
@@ -118,19 +120,14 @@ async function run() {
 
   ammAddress = ret.logs[2].args["0"]
   ammUSDCWBTC = await MinterAmm.at(ammAddress)
+  console.log(`deployed USDC AMM at address ${ammAddress}`)
+  console.log(`USDC AMM uses lpToken: ${await ammUSDCWBTC.lpToken.call()}`)
+
   console.log("completed AMM deploy")
 
   // set deposit limits and whitelist LPs
   await ammWBTCUSDC.setEnforceDepositLimits(true, "70000000") // 0.7 BTC ~ $10K
   await ammUSDCWBTC.setEnforceDepositLimits(true, "10000000000") // $10K
-  await ammWBTCUSDC.setCapitalDepositLimit(
```

Comment:
in a previous PR we disable the `setCapitalDepositLimit`, and so I remove these call sites here